runtime/onert/core/include/backend/train/ITrainableTensor.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __ONERT_BACKEND_TRAIN_ITRAINABLE_TENSOR_H__
+#define __ONERT_BACKEND_TRAIN_ITRAINABLE_TENSOR_H__
```

Comment:
```suggestion
#ifndef __ONERT_BACKEND_TRAIN_I_TRAINABLE_TENSOR_H__
#define __ONERT_BACKEND_TRAIN_I_TRAINABLE_TENSOR_H__
```

Reply:
There are many styles in onert.
`__ONERT_BACKEND_TRAIN_I_TRAINABLE_TENSOR_H__`
`__ONERT_BACKEND_TRAIN_ITRAINABLE_TENSOR_H__`
`__ONERT_BACKEND_TRAIN_ITRAINABLETENSOR_H__`

I don't know which one to follow.
If there is a guide, I will follow that guide in the future.

Reply:
I only knew the `_I_` styles from onert.

e.g.
- ITensor.h - #ifndef `__ONERT_BACKEND_OPERAND_I_TENSOR_H__` (? old days)
- ICompiler.h - #ifndef `__ONERT_COMPILER_I_COMPILER_H_` (by @hseok-oh)

But I searched after you comment above, then I've found other styles. : )

- IConfig.h - #ifndef `__ONERT_BACKEND_ICONFIG_H__` (? old days)
- IOperation.h - #ifndef `__ONERT_IR_IOPERATION_H__` by you (@ragmani)

It seems the style is mixed. Then, it is okay to keep your style. : )